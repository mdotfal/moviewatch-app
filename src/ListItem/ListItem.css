img {
  height: 50px;
}

.btn {
  margin: 2px;
}

h2 {
  font-size: 30px;
}

