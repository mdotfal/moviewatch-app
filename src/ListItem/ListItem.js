import React from 'react';

const ListItem = ({ item }) => {
  
  return (
    <section>
      <header>
        <h2>{ item.title } - <span className='green'>{ item.rating }</span></h2>
      </header>
      <div className='stream-buttons'>
  
      </div>
      <button>Edit</button>
      <button>Delete</button>
    </section>
  );
};

export default ListItem;