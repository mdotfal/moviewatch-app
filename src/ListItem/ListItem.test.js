import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ListItem from './ListItem';

describe.skip( `ListItem component`, () => {
  it( `renders empty given no item`, () => {
    const wrapper = shallow( <ListItem /> )
    expect( toJson( wrapper )).toMatchSnapshot()
  })

  it( `renders the item when supplied`, () => {
    const testItem = { title: 'test-title' }
    const wrapper = shallow( <ListItem item={ testItem }  /> )
    expect( toJson( wrapper ) ).toMatchSnapshot()
  })

})