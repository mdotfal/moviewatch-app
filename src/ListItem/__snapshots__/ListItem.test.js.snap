// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem component renders the item when supplied 1`] = `
<section>
  <p>
    Available on:
  </p>
  <header>
    <h2>
      Test title
       -
      <span>
        <button
          className="red"
        />
      </span>
    </h2>
  </header>
  <div
    className="stream-buttons"
  />
  <button>
    Edit
  </button>
  <button>
    Delete
  </button>
</section>
`;
